__tests__/app.test.js

```diff
@@ -31,8 +32,60 @@ describe("GET /api/topics", () => {
       });
   });
   test("404: should respond with 'Not found' for invalid endpoint", () => {
+    return request(app).get("/api/banana").expect(404);
+  });
+});
+describe("GET /api", () => {
+  test("200 should respond with a JSON object of available endpoints", () => {
     return request(app)
-      .get("/api/banana")
-      .expect(404)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
```

Comment:
this isn't dynamic - what about when you add more info to the json like /api/articles/id . how can we change this so we wouldn't have to change this test?